api/v1beta1/kafkacluster_types.go

```diff
@@ -318,6 +322,12 @@ type IstioIngressConfig struct {
 	VirtualServiceAnnotations map[string]string   `json:"virtualServiceAnnotations,omitempty"`
 	// Envs allows to add additional env vars to the istio meshgateway resource
 	Envs []corev1.EnvVar `json:"envs,omitempty"`
+	// If specified and supported by the platform, this will restrict traffic through the cloud-provider
+	// load-balancer will be restricted to the specified client IPs. This field will be ignored if the
```

Comment:
**Recommended**: we **SHOULD** improve phrasing for clarity. We **SHOULD** also regenerate the CRDs, samples, etc. if changed.

```suggestion
	// If specified and supported by the platform, traffic through the 
	// cloud-provider load-balancer will be restricted to the specified client
	// IPs. This field will be ignored if the
```